packages/instrumenter/src/frameworks/angular-ignore.ts

```diff
@@ -0,0 +1,78 @@
+import type { NodePath as BabelNodePath } from '@babel/core';
+
+import { Ignorer, NodePath } from '@stryker-mutator/api/ignore';
+
+declare module '@stryker-mutator/api/ignore' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface NodePath extends BabelNodePath {}
+}
+
+const ANGULAR_SIGNAL_IO_FUNCTIONS = ['input', 'model', 'output'];
+
+export class AngularIgnore implements Ignorer {
+  public shouldIgnore(path: NodePath): string | undefined {
+    return this.isObjectInsideIOFunction(path)
+      ? 'Angular signal based input, model and output functions configuration object cannot be mutated as that causes issues with the ivy compiler.'
+      : undefined;
+  }
+
+  private isSignalIOFunction(path: NodePath): boolean {
+    if (!path.isCallExpression()) {
+      return false;
+    }
+
+    const isRequiredSignalIOFunction =
```

Comment:
Instead of specifying the exact api's inside `input`, `output`, etc to ignore, why not simply ignore everything inside `input`, `output` and `model` function calls? For safety purposes we could add a check that they appear inside class properties.